_data/privacy-policy.yml

```
@@ -30,19 +30,36 @@
         <ul>
           <li>Device, internet, and mobile information such as the hardware model of the device you are using, the version of the operating system you are using, your unique device identifiers, your browser type, your language choices, your wireless network, and your mobile network information (including your mobile phone number);</li>
           <li>Location-related information, which we derive from things like your device’s IP address or your mobile device’s GPS signal;</li>
-          <li>Information about your visit to our Website or Services, such as the date and time of your visit, the information you searched for to find our Website, the website you came from, the website you visit when you leave us, and how you used or didn’t use our Services; </li>
+          <li>Information about your visit to our Site or Services, such as the date and time of your visit, the information you searched for to find our Site, the website you came from, the website you visit when you leave us, and how you used or didn’t use our Services; </li>
           <li>Information about your activity on our Services such as which modules you chose to view and which modules you chose to deploy;  </li>
-          <li>Errors, crash reports, and other information relating to technical problems on our Website or Services, including basic information about what happened internally leading up to the crash </li>
+          <li>Errors, crash reports, and other information relating to technical problems on our Site or Services, including basic information about what happened internally leading up to the crash.</li>
         </ul>
         <p>Some of this information may be collected using cookies and similar technologies. To learn more about our use of cookies, please see the section entitled “We use cookies and related technologies” below. </p>
         <h3>Information We Collect From Other Sources</h3>
-        <p>We may also collect information from third parties, such as our service providers, when they are legally allowed to share your information with us. For instance, in some circumstances we may receive limited information about your payment card from our payment processor, such as the last four digits of your card, the country of issuance, and the expiration date.</p>
+        <p>We may also collect information from third parties, such as our service providers, when they are legally allowed to share your information with us. For instance, we may collect personal data about you from the following sources: </p>
+        <ul>
+          <li>
+          <u>Payment processors</u> - In general, your payment card information is provided directly to our third-party payment processor and we do not collect or store that information. However, we may receive limited information from our payment processor such as the last four digits of your card, the country of issuance, and the expiration date. 
```

Recommend using an em dash (&mdash;) here instead (and remaining bullets)
```suggestion
          <u>Payment processors</u> &mdash; In general, your payment card information is provided directly to our third-party payment processor and we do not collect or store that information. However, we may receive limited information from our payment processor such as the last four digits of your card, the country of issuance, and the expiration date. 
```